app/models/pydantic/jobs.py

```diff
@@ -138,9 +139,9 @@ class PixETLJob(Job):
 
 
 class GDALCOGJob(Job):
-    """Use for creating COG files using GDAL Python docker in PixETL queue."""
+    """Use for creating COG files using GDAL Python docker in COG queue."""
 
-    job_queue = PIXETL_JOB_QUEUE
+    job_queue = COGIFY_JOB_QUEUE
```

Comment:
The terraform and code looks good, but I don't think we want to name this all specifically as "cogify". That's just one use case for this on demand queue, but we were also interested in using it eventually for other big jobs that often have issues on spot (e.g. like the TCL tile caches). 

So maybe just something like "on-demand-computing" or something? I guess we also have a "pixetl", we should label that more generically at some point too.

Reply:
Good call,  the pixetl one had thrown me off into thinking the queue is coupled with the specific compute environment.